alvr/client_openxr/src/stream.rs

```diff
@@ -548,7 +548,21 @@ fn stream_input_loop(
                 &int_ctx.face_sources,
                 stage_reference_space,
                 now,
-            ),
+            )
+            .map(|option| {
+                option.map(|pose| {
+                    if int_ctx.platform.is_pico() {
+                        let head_rot_inv = int_ctx.relative_head_orientation.conjugate().inverse();
+
+                        Pose {
+                            orientation: head_rot_inv * pose.orientation,
+                            position: head_rot_inv * pose.position,
+                        }
```

Comment:
For simplicity, instead of storing `relative_head_orientation` you could store the whole Pose. This allows you to directly use the Pose * Pose operator here

Reply:
This of course if the bug affects the whole pose, not just the orientation. but in the end we don't care much, as we just use the orientation for VRChat. Do the change anyway as it would simplify the code a bit